cev_eris.dme

```diff
@@ -2441,6 +2441,7 @@
 #include "code\modules\projectiles\guns\projectile\dartgun.dm"
 #include "code\modules\projectiles\guns\projectile\flaregun.dm"
 #include "code\modules\projectiles\guns\projectile\shotgun.dm"
+#include "code\modules\projectiles\guns\projectile\shotgun\ks23.dm"
 #include "code\modules\projectiles\guns\projectile\sniper.dm"
```

Comment:
```suggestion
#include "code\modules\projectiles\guns\projectile\sniper.dm"
#include "code\modules\projectiles\guns\projectile\shotgun\ks23.dm"
```
Wait shouldnt this be a line lower as well?
Did you mannully put this in or did DM mess it up on you?